catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transformer/TransformerManager.java

```diff
@@ -11,10 +11,12 @@
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.spatial.ogc.csw.catalog.transformer;
+package org.codice.ddf.spatial.ogc.csw.catalog.common.transformer;
```

Comment:
It needed to be moved so as to be accessible without the need to embed the transformer bundle. It doesn't have any dependencies other than java and osgi so that is why no dependencies changed.